python/vyos/ifconfig/interface.py

```diff
@@ -1373,7 +1416,7 @@ def _create(self):
             cmd += ' egress-qos-map {egress_qos}'
 
         self._cmd(cmd.format(**self.config))
-
+        
```

Comment:
Please do not add empty whitespace lines.

Reply:
done